iree/tools/utils/cpu_features.c

```diff
@@ -0,0 +1,166 @@
+// Copyright 2022 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/tools/utils/cpu_features.h"
+
+#if defined __linux__ && defined __aarch64__
```

Comment:
Many of these are also defined in https://github.com/google/iree/blob/main/iree/base/target_platform.h, for IREE code to use (`IREE_PLATFORM_LINUX`, `IREE_ARCH_*`). The direct checks as you have them _may_ be more appropriate in this file as written, but the IREE versions are nice to use for consistency.

Reply:
thanks!